src/include/daos/drpc_modules.h

```diff
@@ -33,43 +33,43 @@ enum drpc_sec_agent_method {
 };
 
 enum drpc_mgmt_method {
-	DRPC_METHOD_MGMT_KILL_RANK		= 201,
```

Comment:
Looks like just whitespace changes. Mind reverting this file?

Reply:
done